ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon__v4.13.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rhods-addon-tests:
+    name: ods-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhods-tests
```

Comment:
the naming should be based on https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/
see https://github.com/openshift/release/pull/40239 for examples